bucket/scoop-completion.json

```
@@ -16,21 +16,21 @@
     "post_install": [
         "Get-ChildItem -Path \"$dir\" -Filter \"scoop-completion-$version\" -Directory | Remove-Item -Force -Recurse",
         "Import-Module scoop-completion",
-        "$ScriptsDir = $(Join-Path $(Find-BucketDirectory -Root -Name extras) scripts/scoop-completion)",
-        "$ModifierPath = $(Join-Path $(Find-BucketDirectory -Root -Name extras) scripts/ModifyPSProfile.psm1)",
-        "'append-to-profile.ps1', 'remove-from-profile.ps1' | ForEach-Object {",
-        "    if ((Test-Path \"$ScriptsDir\\$_\") -and (Test-Path \"$ModifierPath\")) {",
-        "        Copy-Item \"$ScriptsDir\\$_\" \"$dir\\\"",
-        "        $Content = (Get-Content \"$dir\\$_\").Replace('$ModifyPSProfile', \"$ModifierPath\")",
-        "        $content | Set-Content -Path \"$dir\\$_\"",
-        "    } else {",
-        "        Write-Host 'Missing files, please update scoop buckets and reinstall this module again' -ForegroundColor Red",
-        "    }",
+        "$BucketDir = Find-BucketDirectory -Root -Name extras",
+        "$UtilsPath = $BucketDir | Join-Path -ChildPath \"scripts\\ModifyPSProfile.psm1\"",
```

```suggestion
        "$UtilsPath = $BucketDir | Join-Path -ChildPath \"scripts\\ModifyPSProfile.psm1\"",
        "Unblock-File $UtilsPath",
```
This can avoid some permission issues